strings/test.js

```diff
@@ -46,15 +47,18 @@ describe('Strings - basics', () => {
     });
   });
 
-  xdescribe('whereIsCharacter', () => {
+  describe('whereIsCharacter', () => {
     it('should return an int value for the location of the character in the string', () => {
       expect(whereIsCharacter('hello', 'e')).toEqual(1);
+      expect(whereIsCharacter('hello', 'f')).toEqual(-1); // not sure this is necessary?
```

Comment:
Very very good - That is correct yeah.

You could take this 1 step further and modify your test to say "I expect this to return false if the character doesn't exist in the string" (instead of it returning -1) then update your code accordingly

Reply:
is -1 falsy or truthy?